### ISSUE TITLE
Translation returned by missing handler is not interpolated

### ISSUE THREAD

**Issue by fracz** [kazupon/vue-i18n#313]
This is a follow up of #297.

Now, we are able to return a dynamic translation from `missing` handler (thanks!), but if it contains variables, they don't get interpolated.

Example: `{{ $t('Some {var}', {var: 'test'}) }}`

My missing handler: `missing(locale, key) { return key; }`

If the `Some {var}` translation is missing, the `missing` handler is called correctly but the result on a page is `Some {var}` instead of `Some test`.

**Comment by virginiogl** [comment]
Same issue
http://jsfiddle.net/qegsr2pn/63/

**Comment by kazupon** [comment]
In Vue I18n, **path** is not interpolated...

**Comment by H3xx3n** [comment]
Yeah but this is clearly an issue. Do you have any clue how to resolve this or maybe a workaround?

**Comment by fracz** [comment]
@kazupon I don't understand what do you mean by "path is not interpolated". Could you explain a bit more why do you think this issue should be closed?

**Comment by kazupon** [comment]
@fracz 
The structure of VueI18n localization message is basically object (dictional) structure as like rails-i18n.
https://kazupon.github.io/vue-i18n/guide/messages.html#structure

Therefore, when we are translating, specify the `path` as follows:

```json
{
   "foo": {
      "bar": "hello {you}!",
      ...
    }
 }
```
```js
$t('foo.bar', { you: 'kazupon' })
```

So, In VueI18n, It does not assume how to use about path interpolating.
I cannot understand why complementing the path. I would like to know about the use-cases. 🤔

**Comment by fracz** [comment]
Our team found the paths very tedious to maintain. While we are aware that paths can solve some amibiguities in translations, we decided to use phrases in english instead of them and translate it to other languages in the language files. This way, the HTML templates of components are so much easy to read and understand.

This works perfectly for messages without an interpolation thanks to the https://github.com/kazupon/vue-i18n/issues/297. We have configured the MissingHandler to return exacly what was passed to `$t`. Example:

```
// template:
{{ $t('Some uninterpolated message') }}
// english language file:
{}
// polish language file:
{"Some uniterpolated message": "Jakaś wiadomość bez interpolacji"}
// result in enlish:
 Some uninterpolated message
// result in polish:
Jakaś wiadomość bez interpolacji
```
However, when the message contains an interpolation, the value of `$t('Some {var}', {var: 'test'})` is displayed literally as `Some {var}` because values returned from missing handler do not have the variabled interpolated. When I add the following entry in english language file `{"Some {var}": "Some {var}"}`, it works, but such duplication does not make any sense.
